.eslintignore

```diff
@@ -1,6 +1,5 @@
 **/node_modules/**
 **/third_party/**
-**/generated/**
```

Comment:
drive-by removal of `generated` which was actually cut before going to master

Reply:
Looks like this was hiding eslint errors in `lighthouse-core/lib/cdt/generated/SourceMap.js`. Maybe just make this rule specific to that directory?